lib/rules/sotd/pp.js

```diff
@@ -21,7 +24,7 @@ function buildWanted (groups, sr) {
 
         var noRec = ((groups.length == 2) ? "The groups do not " : "The group does not ") +
             "expect this document to become a W3C Recommendation\\. ?";
-        if (config.recTrackStatus && config.noRecTrack)
+        if ((config.recTrackStatus && config.noRecTrack )|| sr.isIGDeliverable)
```

Comment:
`isIGDeliverable` doesn't need to be global. It's used only in that file.

Reply:
> I'm still not getting how the test reflects what the rules is supposed to check.

In 'test/docs/headers/wg-note2.html' I add a paragraph, saying the document is delivered by '40318', '109611' two groups. The second id '109611' is a **Interest Group**, that means the sentence `noRec` should be in PP paragraph.

The paragraph pp.js rule is looking for:
> /^This document was produced by a group operating under the( 15 September 2020| 1 August 2017)? W3C Patent Policy\. ?The group does not expect this document to become a W3C Recommendation\. ?$/

But in 'test/docs/headers/wg-note2.html' there is no such paragraph. So in 'test/rules.js' I add error "sotd.pp.no-pp" when checking 'wg-note2.html'.

Reply:
> But in 'test/docs/headers/wg-note2.html' there is no such paragraph. So in 'test/rules.js' I add error "sotd.pp.no-pp" when checking 'wg-note2.html'.

The sentence is actually in that document but in a separate paragraph. The current test is confusing. I believe wg-note2.html was create to make sure both sentences are in the same paragraph.
Now, it's also mixing what we expect from an IG so in the end, we don't really know what we are testing.